controllers/rhmi/rhmi_controller.go

```diff
@@ -78,13 +84,35 @@ const (
 	installTypeEnvName               = "INSTALLATION_TYPE"
 	priorityClassNameEnvName         = "PRIORITY_CLASS_NAME"
 	managedServicePriorityClassName  = "rhoam-pod-priority"
+	alertManagerRouteName            = "alertmanager-route"
+	routeRequestUrl                  = "/apis/route.openshift.io/v1"
 )
 
 var (
 	productVersionMismatchFound bool
 	log                         = l.NewLoggerWithContext(l.Fields{l.ControllerLogContext: "installation_controller"})
+	alertsToSilence             = []string{
+		"KeycloakInstanceNotAvailable",
+	}
 )
 
+type Silence struct {
```

Comment:
Minor: this type can be imported from here I believe:
https://github.com/prometheus/alertmanager/blob/22ac6dff21901bfce14545da59b37a1aaca0db3a/api/v2/models/silence.go#L32-L55

Reply:
imported, thanks :D